### PR TITLE
Source MySQL: Tagging MySQL variants as GA in metadata. Prep for Airbyte Certified/Bronze release

### PR DESCRIPTION
**Pull request by prateekmukhedkar**
Refer #28681, tagging as GA for internal visibility.

### CODE REVIEW

**Comment by bnchrch** [comment]
@prateekmukhedkar Ill leave it to your team for the approval but I just updated the internal metadata to match
https://www.notion.so/Internal-Metadata-Fields-32b02037e7b244b7934214019d0b7cc9?pvs=4

**Review by bnchrch — COMMENTED** [review]
Ill leave the approval to the db sources team :) 

But I updated the internal metadata fields to match the new support level!

https://www.notion.so/Internal-Metadata-Fields-32b02037e7b244b7934214019d0b7cc9?pvs=4

File: airbyte-integrations/connectors/source-mysql/metadata.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 data:
   ab_internal:
     ql: 300
-    sl: 100
+    sl: 200
```
Comment (prateekmukhedkar) [inline airbyte-integrations/connectors/source-mysql/metadata.yaml:4]: oh, it shoud be 300 then. we are certifying as Bronze. Updated PR title to be clear.

**Comment by prateekmukhedkar** [comment]
> Ill leave the approval to the db sources team :)
> 
> But I updated the internal metadata fields to match the new support level!
> 
> https://www.notion.so/Internal-Metadata-Fields-32b02037e7b244b7934214019d0b7cc9?pvs=4

@bnchrch we want to publicly announce source-mysql as Airbyte certified. Should we bump `ql` to 300?

**Comment by bnchrch** [comment]
@prateekmukhedkar Feel free to! Its not required at this time but worth to point out that bumping of `ql` will increase the number of test cases that we certify `source-mysql` against.

In this case going to `ql: 300` will require that you define the `allowedHosts` field in the metadata

Which it looks like you have! so I'd say there is no harm

**Review by akashkulk — APPROVED** [review]

**Pull request merged**